packages/core/src/meta/ruleset.schema.json

```diff
@@ -114,8 +117,22 @@
         ]
       },
       "errorMessage": {
-        "minItems": "must not be empty",
-        "type": "must be an array"
+        "minItems": "must not be empty"
+      }
+    },
+    "aliases": {
+      "type": "object",
+      "propertyNames": {
+        "pattern": "^[A-Za-z][A-Za-z0-9_-]*$",
```

Comment:
🤔 Pattern aliases would also be cool, so you could have a re-usable pattern called `c-style-identifier` etc.